python/mxnet/ndarray.py

```diff
@@ -1015,24 +1015,23 @@ def full(shape, val, ctx=None, dtype=mx_real_t):
 
 
 def array(source_array, ctx=None, dtype=None):
-    """Creates a new array from any object exposing the array interface.
+    """Creates an array from any object exposing the array interface.
 
     Parameters
     ----------
     source_array : array_like
-        Any object exposing the array interface, an object whose ``__array__``
+        An object exposing the array interface, an object whose `__array__`
         method returns an array, or any (nested) sequence.
     ctx : Context, optional
-        An optional device context (default is the current default context).
+        Device context (default is the current default context).
     dtype : str or numpy.dtype, optional
-
-        An optional value type. If the ``source_array`` is an NDArray, then defaults to
-        ``source_array.dtype``, otherwise default to ``float32``.
+        The data type of the output array. The default dtype is ``source_array.dtype``
+        if `source_array` is an NDArray, `float32` otherwise.
 
     Returns
     -------
     NDArray
-        An ``NDArray`` array with the same contets as the ``source_array``.
+        An `NDArray` satisfying the requirements specified by the parameters.
```

Comment:
I already reviewed this line before. Are multiple people working on the same APIs or were previous suggestions not implemented?